README.md

```diff
@@ -1,23 +1,27 @@
 # README
 
-## Rails Type
-
-This app uses Rails 5
-
 ## What is It?
 
-This app is a database of Special Transit passengers, to be used by ST dispatchers and administrators.
+This app is a database of Special Transit passengers, to be used by ST
+dispatchers and administrators.
 
 ## What Does It Do?
 
-- Stores all Spacial Transit passengers and their attributes. This allows for functionality such as automatically deactivating temporary ST passengers who have expired doctors notes (done with a cron job that runs every day at 4:30am)
+Stores all Spacial Transit passengers and their attributes. This allows for
```

Comment:
Who are we, [Blue Origin](https://www.nytimes.com/2021/05/05/science/blue-origin-space-jeff-bezos.html)?